src/generic_ui/polymer/invite-user.html

```diff
@@ -202,7 +202,7 @@
     <uproxy-state id="state"></uproxy-state>
     <core-signals on-core-signal-open-invite-user-dialog='{{ openInviteUserPanel }}'></core-signals>
 
-    <core-overlay id='inviteUserPanel'>
+    <core-overlay id='inviteUserPanel' transition='none'>
```

Comment:
(note, the correct spelling for this is actually `transition=''`, as amended in 73dae97. (still didn't fix the janky + animation, as noted in this PR's comments))